pkg/util/rand/rand.go

```diff
@@ -32,6 +32,16 @@ var rng = struct {
 	rand: rand.New(rand.NewSource(time.Now().UTC().UnixNano())),
 }
 
+func Int(max int) int {
+	if max <= 0 {
```

Comment:
https://github.com/jayunit100/kubernetes/blob/backoffresets/pkg/util/rand/rand.go#L47 seems to also be doing this.
Should I consolidate these two functions + remove the panics?